src/vulkan-renderer/application.cpp

```diff
@@ -395,9 +395,9 @@ VkResult Application::init() {
 
     spdlog::debug("Creating Vulkan instance.");
 
-    result = create_vulkan_instance(application_name, engine_name, application_version, engine_version, enable_khronos_validation_instance_layer,
-                                    enable_renderdoc_instance_layer);
-    vulkan_error_check(result);
+    // Create a Vulkan instance.
+    vkinstance = std::make_unique<wrapper::Instance>(application_name, engine_name, application_version, engine_version, VK_API_VERSION_1_1, std::nullopt,
+                                                     std::nullopt, true, false);
 
 #if defined(_DEBUG)
```

Comment:
Should be `#ifndef NDEBUG` to be cross platform compatible.

Reply:
You mean `#ifdef NDEBUG` :D

Reply:
done.

Reply:
No, it should be `#ifndef NDEBUG`

Reply:
I fixed another macro which had this issue.